velox/common/caching/SsdFile.h

```diff
@@ -411,6 +411,9 @@ class SsdFile {
 
   // True if there was an error with checkpoint and the checkpoint was deleted.
   bool checkpointDeleted_{false};
+
+  // True if copy on write should be disabled.
+  bool disableFileCow_{false};
```

Comment:
This should be a const? The same for checkpointIntervalBytes_? And put the const members first? Thanks!

Reply:
`checkpointIntervalBytes_` should not be const as it needs to be set to `0` when checkpoint error happens (to disable checkpoint).